crates/codegen/schema/src/utils.rs

```diff
@@ -8,11 +8,10 @@ impl Grammar {
     pub fn collect_version_breaks<'a>(&'a self) -> BTreeSet<Version> {
         let mut version_breaks = BTreeSet::new();
         version_breaks.insert(self.versions.first().cloned().unwrap());
```

Comment:
We shouldn't put the last version here, as the "last" version as far is it is concerned is the highest one already specified on one of the `versioned:` nodes, not the last version in the manifest file (not necissarily used).